crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/RunSavedSearchCommand.java

```diff
@@ -70,15 +66,18 @@ public RunSavedSearchCommand(String searchName,
         this.from=from;
         this.to=to;
         this.params=properties;
+        this.metrics = metrics;
     }
 
     @Override
     public CallStatus run() {
+        RequestMetrics.Context metricCtx = metrics.startSavedSearchRequest("savedsearch", searchName, entity, version);
```

Comment:
I think passing the "savedsearch" argument is not needed now (we know it is a saved search already).

Reply:
Forgot to remove the extra param. Thanks for catching this one. Fixed.